Makefile

```diff
@@ -1,19 +1,27 @@
+CONTAINER_NAME=packit
+CONTAINER_EXEC=podman exec $(CONTAINER_NAME)
+
 prepare-check:
 	ansible-playbook -b -K -i inventory-local -c local ./recipe-tests.yaml
 
-check:
-	tox
+container:
+	rsync -a $(CURDIR)/ /tmp/packit
+	podman container inspect packit || podman run --name $(CONTAINER_NAME) -ti -d -v /tmp/packit:/src:Z -w /src fedora:30
```

Comment:
It would be also cleaner to have a dedicated image (with a dockerfile) where the deps would get installed.

Reply:
yep could be, but this also solves issue, when fedora:X image is updated, so that I don't need to regenerate and upload image somewhere